INSTALL

```diff
@@ -36,6 +36,7 @@ Contents
               python-4suite-xml python-simplejson python-xml \
               gnuplot poppler-utils \
               gs-common clisp gettext libapache2-mod-wsgi unzip \
+              pkg-config \
```

Comment:
BTW this `aptitude` string is obsolete in the `next` branch; it is a remainder from the `master` branch.
(Even there, in the `master` branch, it is recommended to use `pip` rather than system packages.)